storage/users.go

```diff
@@ -9,35 +9,77 @@ import (
 	"github.com/Southclaws/cj/types"
 )
 
-// User is a recorded and verified SA:MP forum user.
+// User is a recorded and verified burgershot forum user.
 type User struct {
-	DiscordUserID    string `json:"discord_user_id" bson:"discord_user_id"`
-	ForumUserID      string `json:"forum_user_id" bson:"forum_user_id"`
-	VerificationCode string `json:"verification_code" bson:"verification_code"`
-	ForumUserName    string `json:"forum_user_name" bson:"forum_user_name"`
+	DiscordUserID string `json:"discord_user_id" bson:"discord_user_id"`
+	ForumUserID   string `json:"forum_user_id" bson:"forum_user_id"`
+	ForumUserName string `json:"forum_user_name" bson:"forum_user_name"`
+	BurgerVerify  bool   `json:"burgershot_verified" bson:"burgershot_verified"`
```

Comment:
This is missing a `BurgerUserID` field - don't want to re-use the SA:MP forum user ID field because we'd lose existing links.

Reply:
What should we do about /whois? Should we just direct it to using the Burger forum ID?